.github/workflows/label-pr.yml

```diff
@@ -1,18 +0,0 @@
-name: Label PRs
```

Comment:
I don't think we should remove this one. It's better to just create a new workflow completely separated from `label-pr`.